### DISCUSSION TITLE
Is appflowy running stably now?

### DISCUSSION THREAD

**Discussion by naughtz** [AppFlowy-IO/AppFlowy#151] [General]
I want to use appflowy to document my work. Would appflowy keep downward compatibility? I'm worried about losing data.

**Comment by MikeWallaceDev** [comment]
I'm curious to hear what @appflowy has to say about this?

**Comment by annieappflowy** [comment]
> I want to use appflowy to document my work. Would appflowy keep downward compatibility? I'm worried about losing data.

Thanks for asking. V 0.1 and beyond will have downward compatibility. 

We're now at V 0.0.2 
We can build an exporter for versions under v 0.1 to migrate user data.
